.github/workflows/msbuild.yml

```diff
@@ -47,9 +47,19 @@ jobs:
         cp -r z3* dafny/Binaries/z3
     - name: Nuget Restore Dafny
       run: nuget restore dafny/Source/Dafny.sln
+    - name: Build Dafny Parser
+      run: make -C dafny/Source/Dafny -f Makefile.Linux
     - name: Build Dafny
       run: msbuild dafny/Source/Dafny.sln
     - uses: actions/setup-python@v1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build DafnyRuntime.jar with Gradle
+      run: dafny/Source/DafnyRuntime/DafnyRuntimeJava/gradlew -p dafny/Source/DafnyRuntime/DafnyRuntimeJava build
```

Comment:
Same comment here, also it depends on whether `copyJarToBinaries` declares its dependencies correctly

Reply:
Added the dependency and edited correspondingly.